hr_expense_invoice/models/hr_expense.py

```diff
@@ -205,4 +178,3 @@ def _compute_amount_residual(self):
                 lambda x: x.account_type in ("asset_receivable", "liability_payable")
             )
             rec.amount_residual = -sum(payment_term_lines.mapped(residual_field))
-        return super(HrExpense, self - applicable_expenses)._compute_amount_residual()
```

Comment:
The super call should never be removed, as we are breaking the compatibility with other possible extensions.